sdk/monitor/opentelemetry-exporter-azure-monitor/test/unit/platform/nodejs/httpSender.test.ts

```diff
@@ -54,11 +55,19 @@ describe("HttpSender", () => {
     });
 
     it("should send a partially retriable envelope", async () => {
-      const sender = new HttpSender();
+      const sender = new HttpSender(DEFAULT_EXPORTER_CONFIG);
       scope.reply(206, JSON.stringify(partialBreezeResponse([200, 408, 408])));
       const { result, statusCode } = await sender.send([envelope, envelope]);
       assert.strictEqual(statusCode, 206);
       assert.deepStrictEqual(JSON.parse(result), partialBreezeResponse([200, 408, 408]));
     });
+
+    it("should build endpoint using config", async () => {
+      const sender = new HttpSender({
+        endpointUrl: "https://test.com",
```

Comment:
What happens if I manage to get 'https://test.com/' in there as the endpointUrl? Does it trim the extra end slash?